vllm/engine/arg_utils.py

```diff
@@ -932,6 +934,15 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             default="auto",
             help='The worker class to use for distributed execution.')
 
+        parser.add_argument(
+            "--generation-config",
+            type=nullable_str,
+            default=None,
+            help="The folder path to the generation config. If set to "
+            "'auto', the generation config will be automatically loaded "
+            "from the model's folder.",
```

Comment:
Can you explain the difference between `None` and `"auto"` in the help text?

Reply:
Done